viite-backend/viite-main/src/main/scala/fi/liikennevirasto/viite/ProjectService.scala

```diff
@@ -1580,45 +1580,71 @@ def setCalibrationPoints(startCp: Long, endCp: Long, projectLinks: Seq[ProjectLi
         logger.warn(s"Ely-codes for project: $projectId were not updated.")
     }
 
+
+    /**
+     * Updates project links with data from the dev tool.
+     *
+     * This function takes the manually given devToolData, which contains the new address, roadway, and side code information,
+     * and updates the corresponding project links with these values. It also handles the creation of new
+     * roadway numbers if necessary, adjusts status and side codes, and sets calibration points for roadways.
+     *
+     * The function performs the following steps:
+     * 1. If both start and end address M values are defined, it updates the project links with these address values.
```

Comment:
"it updates the project links with these address values"

"the project links" is not clear from the context.
If referring to the projectLinks parameter, use that exact name:
"it updates projectLinks with these address values"

Reply:
(Obsolete :) As the others, too. Resolving.)